### PR TITLE
Readme update for redactor functionality [#22]

### PR DESCRIPTION
**Pull request by red1408**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +109,22 @@ env:
   value: *containerName # referencing earlier anchor in same yaml
 ```
 
+## DICOM Redactor
+
+Import adapter can be configured to use [DICOM Redactor Library](https://github.com/GoogleCloudPlatform/healthcare-deid/tree/master/offline_tools/redactorhttps://github.com/GoogleCloudPlatform/healthcare-deid/tree/master/offline_tools/redactor) to redact sensitive data contained in DICOM tags during C-STORE upload.
```
Comment (danielbeaudreau) [inline README.md]: Phrasing edit: "The import adapter can be configured to use the [DICOM Redactor Library]"
... "during a C-STORE upload."

File: README.md [hunk 2]
```diff
@@ -109,6 +109,22 @@ env:
...
+- redact_remove_list - a list of DICOM tags to remove. Other tags are kept untouched.
+- redact_filter_profile - a predefined profile that will keep and remove particular tags.
+
+If enabled by setting one of above options, redactor also always regenerates the following UIDs:
+- StudyInstanceUID
+- SeriesInstanceUID
+- SOPInstanceUID
+- MediaStorageSOPInstanceUID
+
+Note that current implementation of redactor reads whole instance to be redacted into memory, so it may run into Out Of Memory exceptions on particularly large instances.
```
Comment (danielbeaudreau) [inline README.md]: We can remove this sentence. If users experience OOMs and file bugs we can invest more time into optimizing memory usage.

File: README.md [hunk 3]
```diff
@@ -109,6 +109,22 @@ env:
...
 
+## DICOM Redactor
+
+Import adapter can be configured to use [DICOM Redactor Library](https://github.com/GoogleCloudPlatform/healthcare-deid/tree/master/offline_tools/redactorhttps://github.com/GoogleCloudPlatform/healthcare-deid/tree/master/offline_tools/redactor) to redact sensitive data contained in DICOM tags during C-STORE upload.
+The user can configure which tags to redact/remove in one of 3 ways:
+- redact_keep_list - a list of DICOM tags to keep untouched. Other tags are removed.
+- redact_remove_list - a list of DICOM tags to remove. Other tags are kept untouched.
+- redact_filter_profile - a predefined profile that will keep and remove particular tags.
+
+If enabled by setting one of above options, redactor also always regenerates the following UIDs:
```
Comment (danielbeaudreau) [inline README.md]: Phrasing edit, and added a bit more description about the new value:
"If enabled via one of the above options, the redactor also replaces the following UIDs with a random value:"

File: README.md (hunk 2 above)
Comment (red1408) [reply README.md]: Ok

File: README.md (hunk 1 above)
Comment (red1408) [reply README.md]: Ok

File: README.md (hunk 3 above)
Comment (red1408) [reply README.md]: Ok, but a little nitpick: resulting value is fully determined by old one, at least in current implementation of redactor (and this property is important to integration and unit tests).

File: README.md [hunk 4]
```diff
@@ -109,6 +109,20 @@ env:
   value: *containerName # referencing earlier anchor in same yaml
 ```
 
+## DICOM Redactor
```
Comment (danielbeaudreau) [inline README.md]: Can you please add an entry to the table of contents?

File: README.md [hunk 5]
```diff
@@ -109,6 +109,20 @@ env:
...
+
+Import adapter can be configured to use the [DICOM Redactor Library](https://github.com/GoogleCloudPlatform/healthcare-deid/tree/master/offline_tools/redactorhttps://github.com/GoogleCloudPlatform/healthcare-deid/tree/master/offline_tools/redactor) to redact sensitive data contained in DICOM tags during a C-STORE upload.
```
Comment (danielbeaudreau) [inline README.md]: nitpick: 'The Import Adapter' instead of 'Import adapter'

File: README.md (hunk 3 above)
Comment (danielbeaudreau) [reply README.md]: Good point, let's revert it back from:
"the redactor also replaces the following UIDs with a random value:"
to
"the redactor also always regenerates the following UIDs:"

File: README.md (hunk 5 above)
Comment (red1408) [reply README.md]: Ok

File: README.md (hunk 4 above)
Comment (red1408) [reply README.md]: Ok

File: README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (red1408) [reply README.md]: Ok, fixed.

File: README.md (hunk 3 above)
Comment (red1408) [reply README.md]: Ok

**Pull request merged**